### PR TITLE
Update advanced.md

### LINKED ISSUE #13864: Add PR wrangler process to contributor docs
## Feature request

**What to add**

1. Identify process-related content from the wiki page for [PR wranglers](https://github.com/kubernetes/website/wiki/PR-Wranglers).
2. Extract process content from the wiki and add it to the [K8s contributor guidelines](https://kubernetes.io/docs/contribute/).
3. Clean up schedule-related content in the wiki.

**Why is this needed?**

We need to add PR wrangler process content to contributor docs, and make the website's contribution guidelines a single source of truth.

We need to make it easy to update the PR wrangler schedule, and to add schedules for similar roles (new contributor ambassadors, chair shadows, issue triage lead).

**Comments**

Talk with @zacharysarah for clarification as needed.

### PR DESCRIPTION
**Pull request by shavidissa**
Fixes #13864. Organized and merged the content that was originally on this page and on the wiki (https://github.com/kubernetes/website/wiki/PR-Wranglers).

### CODE REVIEW

File: content/en/docs/contribute/advanced.md [hunk 1]
```diff
@@ -19,42 +19,47 @@ client and other tools for some of these tasks.
...
-  - Merge PRs when they are ready, or close PRs that shouldn't be accepted.
-- Triage and tag incoming issues daily. See
-  [Intermediate contributing](/docs/contribute/intermediate/) for guidelines
-  about how SIG Docs uses metadata.
+SIG Docs [approvers](/docs/contribute/participating/#approvers) take regular turns as the PR wrangler for the repository and are added to the [PR Wrangler rotation scheduler](https://https://github.com/kubernetes/website/wiki/PR-Wranglers#2019-schedule-q1q2) for weekly rotations. 
+
+The PR wrangler’s duties include:
+
+- Review [open pull requests](https://https://github.com/kubernetes/website/pulls) daily for quality and adherence to the [style guide](/docs/contribute/style/style-guide/).
+    - Start with XS PRs and head towards XXL PRs (in terms of the PR size).
```
Comment (zacharysarah) [inline content/en/docs/contribute/advanced.md]: Change to:
>     - Review the smallest PRs (`size/XS`) first, then iterate towards the largest (`size/XXL`).

File: content/en/docs/contribute/advanced.md [hunk 2]
```diff
@@ -19,42 +19,47 @@ client and other tools for some of these tasks.
...
 
 The following queries are helpful when wrangling. After working through these three queries, the remaining list of PRs to be
 reviewed is usually small. These queries specifically exclude localization PRs, and only include the `master` branch (except for the last one).
 
 - [No CLA, not eligible to merge](https://github.com/kubernetes/website/pulls?q=is%3Aopen+is%3Apr+label%3A%22cncf-cla%3A+no%22+-label%3Ado-not-merge+label%3Alanguage%2Fen):
-  Remind the contributor to sign the CLA. If they've already been reminded by both the bot and a human, close
+  Remind the contributor to sign the CLA. If they have already been reminded by both the bot and a human, close
   the PR and remind them that they can open it after signing the CLA.
-  **We can't even review PRs whose authors have not signed the CLA!**
+  **Do not review PRs whose authors have not signed the CLA!**
```
Comment (zacharysarah) [inline content/en/docs/contribute/advanced.md]: Nice change! ✨ Definitely more in line with the style guide.

File: content/en/docs/contribute/advanced.md [hunk 3]
```diff
@@ -19,42 +19,47 @@ client and other tools for some of these tasks.
...
 - [Needs LGTM](https://github.com/kubernetes/website/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+-label%3Ado-not-merge+label%3Alanguage%2Fen+-label%3Algtm+):
   If it needs technical review, loop in one of the reviewers suggested by the bot. If it needs docs review
   or copy-editing, either suggest changes or add a copyedit commit to the PR to move it along.
 - [Has LGTM, needs docs approval](https://github.com/kubernetes/website/pulls?q=is%3Aopen+is%3Apr+-label%3Ado-not-merge+label%3Alanguage%2Fen+label%3Algtm):
-  See if you can figure out what needs to happen for the PR to be merged.
+  Find out if any additional changes or updates need to be made for the PR to be merged. If you think the PR is ready to be merged, comment `/approve`.
```
Comment (zacharysarah) [inline content/en/docs/contribute/advanced.md]: Change:
> Find out if any additional changes

To:
> Determine whether any additional changes

File: content/en/docs/contribute/advanced.md [hunk 4]
```diff
@@ -19,42 +19,47 @@ client and other tools for some of these tasks.
...
 - [Not against master](https://github.com/kubernetes/website/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+-label%3Ado-not-merge+label%3Alanguage%2Fen+-base%3Amaster): If it's against a `dev-` branch, it's for an upcoming release.
   Make sure the [release meister](https://github.com/kubernetes/sig-release/tree/master/release-team) knows about it.
-  If it's against an old branch, help the PR author figure out whether it's targeted against the best branch.
+  You can let them know by adding a comment with `/assign @<meister's_github-username>`. If it's against an old branch, help the PR author figure out whether it's targeted against the best branch.
```
Comment (zacharysarah) [inline content/en/docs/contribute/advanced.md]: Rework L61-62 slightly:
> Make sure the [release meister](https://github.com/kubernetes/sig-release/tree/master/release-team) knows about it by adding a comment with `/assign @<meister's_github-username>`.
> If it's against an old branch, help the PR author figure out whether it's targeted against the best branch.

**Comment by shavidissa** [comment]
@zacharysarah: Thank you for the review. Updated the content.

**Pull request merged**